t/lib/t/MusicBrainz/Server/Data/Recording.pm

```diff
@@ -5,7 +5,7 @@ use Test::More;
 
 use MusicBrainz::Server::Data::Recording;
 use MusicBrainz::Server::Data::Search;
-use encoding 'utf8';
+use utf8;
```

Comment:
There isn’t actually any non-ASCII text in this file, but perhaps someone will add UTF-8 test data later … In any case, `use encoding` is deprecated.